cli/lib/kontena/scripts/completer.rb

```diff
@@ -90,7 +90,7 @@ def containers
   def yml_services
     require 'yaml'
     if File.exist?('kontena.yml')
-      yaml = YAML.safe_load(File.read('kontena.yml'))
+      yaml = YAML.safe_load(File.read('kontena.yml'), [], [], true, 'kontena.yml')
```

Comment:
perhaps not very necessary, not very well covered, but i see no way to fail either.